common/defines/babylon_defines.txt

```diff
@@ -11,17 +11,17 @@
 NWar = {
 	DEVASTATION_MAX = 100.0
 	DEVASTATION_INCREASE_RATE = 0.1 # Per day, scales with level of occupation
-	DEVASTATION_DECAY_RATE = -0.06 #-0.1 Per day, when there are no occupations
+	DEVASTATION_DECAY_RATE = -0.07 #-0.1 Per day, when there are no occupations
 	DEVASTATION_FROM_BATTLES = 0.15 # For each battle
 
 	AUTO_CAPITULATE_WAR_SUPPORT = -100 # When war support is this low, the country will auto capitulate
 	DAYS_BETWEEN_WAR_EXHAUSTION = 7 # Every this many days war support will be reduced by the war exhaustion formula
 	WAR_EXHAUSTION_BASE = 0.25
-	WAR_EXHAUSTION_KIA_FACTOR = 5.0 #20 # Scaled by casualties compared to total manpower
-	WAR_EXHAUSTION_KIA_BASE_MULTIPLIER = 0.05 #0.1	# War Exhaustion from casualties is always multiplied by at least this
+	WAR_EXHAUSTION_KIA_FACTOR = 2.0 #20 # Scaled by casualties compared to total manpower
+	WAR_EXHAUSTION_KIA_BASE_MULTIPLIER = 0.04 #0.1	# War Exhaustion from casualties is always multiplied by at least this
```

Comment:
This is a big change, going from 5 to 2. Have you tested this?

Reply:
I tested it with 5 and it was still way too high, and with regards to casualties specifically I would rather it lower than higher importance-wise for war support. 2 is still huge, as at 100% casualties that'd still mean a nation at 100 war support would still drop out of the war in less than a year and a half if there is no wargoal set against them. The goal there is to get war length back to more rational levels, where full occupation and wargoal completion affect the support way more than kia count.